pkg/asset/config/appliance_config.go

```diff
@@ -168,7 +169,17 @@ func (a *ApplianceConfig) Load(f asset.FileFetcher) (bool, error) {
 
 	config := &types.ApplianceConfig{}
 	if err = yaml.UnmarshalStrict(file.Data, config); err != nil {
-		return false, errors.Wrapf(err, "failed to unmarshal %s", ApplianceConfigFilename)
+		// Log full error only on debug level
+		logrus.Debug(err)
+
+		// Search for failed to parse field
+		r := regexp.MustCompile(`field .\S*`)
+		field := r.FindString(err.Error())
+		if field != "" {
+			field = fmt.Sprintf(" (error in %s)", field)
+		}
+		
+		return false, errors.New(fmt.Sprintf("can't parse %s. Ensure the config file is configured correctly%s.", ApplianceConfigFilename, field))
```

Comment:
since we hare hiding the stack trace, please append to this msg that the user may run in debug mode, in case they need more details about the error.
Otherwise LGTM